components/BottomSheet/src/MDCBottomSheetController.h

```diff
@@ -110,6 +110,8 @@
  */
 @property(nonatomic, assign) MDCShadowElevation elevation;
 
+- (nonnull instancetype)init NS_DEPRECATED_IOS(9_0, 9_0, "Use initWithContentViewController:");
```

Comment:
Shouldn't this be `NS_UNAVAILABLE`?

Reply:
After checking internally nobody is using this. But, there is a chance clients are using it via CocoaPods, I think it is better to do deprecated this week and unavailable next what do you think?

Reply:
My understanding is that it always caused a crash in any client that used it.  In that case this is a bug fix and not a breaking change. 

If I've misunderstood the issue, please try to make the referenced Issue and/or PR summary a bit more explicit about the crash conditions.

Reply:
If a client never calls `viewDidLoad` then the crash wouldn't occur. For instance if they were using the `init` in a test file.

```objc
- (void)testFoo {
  MDCBottomSheetController *bottomSheet = [[MDCBottomSheetController alloc] init];

  bottomSheet.elevation = 5;

  XCTAssertEqualWithAccuracy(bottomSheet.elevation, 5, 0.001);
}
```

That would not cause a crash but if they were using it in production to present a bottom sheet then the crash would occur.

Reply:
I think it's fair to assume that being unable to use `init` in production meant that nobody was using it.  I think we should mark `-init` unavailable and consider this to be a non-breaking bug fix PR.  Is that OK with you?

Reply:
Done